tests/test_core.py

```diff
@@ -226,20 +226,20 @@ def test_compute_mean_std_chunked(Q, T):
 
 
 @pytest.mark.parametrize("Q, T", test_data)
-def test_compute_mean_std_chunked_many(Q, T):
+def test_compute_mean_std_chunked(Q, T):
     m = Q.shape[0]
 
-    config.STUMPY_MEAN_STD_NUM_CHUNKS = 128
-    ref_μ_Q, ref_σ_Q = naive_compute_mean_std(Q, m)
-    ref_M_T, ref_Σ_T = naive_compute_mean_std(T, m)
-    comp_μ_Q, comp_σ_Q = core.compute_mean_std(Q, m)
-    comp_M_T, comp_Σ_T = core.compute_mean_std(T, m)
+    config.STUMPY_MEAN_STD_NUM_CHUNKS = 2
+    left_μ_Q, left_σ_Q = naive_compute_mean_std(Q, m)
+    left_M_T, left_Σ_T = naive_compute_mean_std(T, m)
+    right_μ_Q, right_σ_Q = core.compute_mean_std(Q, m)
+    right_M_T, right_Σ_T = core.compute_mean_std(T, m)
     config.STUMPY_MEAN_STD_NUM_CHUNKS = 1
 
-    npt.assert_almost_equal(ref_μ_Q, comp_μ_Q)
-    npt.assert_almost_equal(ref_σ_Q, comp_σ_Q)
-    npt.assert_almost_equal(ref_M_T, comp_M_T)
-    npt.assert_almost_equal(ref_Σ_T, comp_Σ_T)
+    npt.assert_almost_equal(left_μ_Q, right_μ_Q)
+    npt.assert_almost_equal(left_σ_Q, right_σ_Q)
+    npt.assert_almost_equal(left_M_T, right_M_T)
+    npt.assert_almost_equal(left_Σ_T, right_Σ_T)
 
```

Comment:
I seem to have messed up something here, will revert to the form it is in `main` right now